### DISCUSSION TITLE
Sign for Checkout and Checkin asset.

### DISCUSSION THREAD

**Discussion by hungtrungthinh** [snipe/snipe-it#13974] [Ideas]
Hi,
Can we add SIGN feature for users when check-out and check-in assets?
Thank you!

**Comment by PonlorkBeang** [comment]
Dear developers, currently, you create sign feature for Checkout but we also need it for Checkin also. For comply with ISO they need user to confirm when they checked in. This features should include sign from IT user who received the device too.
